content-src/components/Base/Base.js

```diff
@@ -5,6 +5,10 @@ const {actions} = require("actions/action-manager");
 const Header = require("components/Header/Header");
 const TopSites = require("components/TopSites/TopSites");
 const ActivityFeed = require("components/ActivityFeed/ActivityFeed");
+const Spotlight = require("components/Spotlight/Spotlight");
+
+// TODO: replace this with the appropriate actions/reducers/shim.
+const fakeSpotlightItems = require("lib/shim").data.fakeSpotlightItems;
```

Comment:
I wasn't sure if we were going to need new actions or reuse the existing ones so just did this for now...